xcp/net/ifrename/logic.py

```diff
@@ -336,10 +336,8 @@ def rename_logic( static_rules,
     # For completely new network cards which we have never seen before, work out
     # a safe new number to assign it
     ethnumbers = sorted(
-        map(lambda x: int(x[3:]),
-            filter(lambda x: VALID_ETH_NAME.match(x) is not None,
-                   map(lambda x: x.tname or x.kname,
-                       static_rules + cur_state + last_state))))
+        [int(x[3:]) for x in filter(lambda x: VALID_ETH_NAME.match(x) is not None,
```

Comment:
I guess this is the output of the tool, it probably makes more sense to use an `if` instead of the filter

Reply:
Nevermind, it's changed in the successive commit